cluster/kube-down.sh

```diff
@@ -35,6 +35,11 @@ verify-prereqs
 echo "... calling verify-kube-binaries" >&2
 verify-kube-binaries
 echo "... calling kube-down" >&2
-kube-down
+if [[ $PRESET_INSTANCES_ENABLED == $TRUE ]]; then
```

Comment:
I am seeing $TRUE and "true" both being used. Please keep consistent style

Reply:
There are two "true" in file configure-vm.sh. It's the file which will be copied and executed in the preset vm. It doesn't source common.sh so it has no $TRUE. 
The usage in util.sh are all $TRUE.

Reply:
We should keep code style consistent